src/main/drivers/accgyro/accgyro_spi_mpu9250.c

```diff
@@ -50,51 +50,38 @@ static void mpu9250AccAndGyroInit(gyroDev_t *gyro);
 
 static bool mpuSpi9250InitDone = false;
 
-#define DISABLE_MPU9250(spiCsnPin)       IOHi(spiCsnPin)
-#define ENABLE_MPU9250(spiCsnPin)        IOLo(spiCsnPin)
-
-void mpu9250SpiResetGyro(void)
-{
-    // Device Reset
-#ifdef MPU9250_CS_PIN
-    busDevice_t bus = { .spi = { .csnPin = IOGetByTag(IO_TAG(MPU9250_CS_PIN)) } };
-    mpu9250SpiWriteRegister(&bus, MPU_RA_PWR_MGMT_1, MPU9250_BIT_RESET);
-    delay(150);
-#endif
-}
-
 bool mpu9250SpiWriteRegister(const busDevice_t *bus, uint8_t reg, uint8_t data)
 {
-    ENABLE_MPU9250(bus->spi.csnPin);
+    IOLo(bus->spi.csnPin);
     delayMicroseconds(1);
```

Comment:
This delay is probably unnecessary - may be leftover from testing or faulty hardware ... 
MPU datasheet specifies 8ns between CS low and clock low ...